getting_started.Rmd

```diff
@@ -66,23 +66,7 @@ In particular, the rendered preview on pull requests won't work, but you can alt
 
 After creating your repository, you will find that issues related to starting your course have been automatically generated and filed in your repository. To begin the setup process, navigate to the Issues tab and follow the instructions provided in those issues.
 
-
-### 5. Add jhudsl-robot as a collaborator
```

Comment:
This is a parallel change with https://github.com/jhudsl/OTTR_Template/pull/728/